src/test/java/org/broadinstitute/hellbender/tools/spark/pipelines/ReadsPipelineSparkIntegrationTest.java

```diff
@@ -121,6 +122,7 @@ public void testReadsPipelineSpark(PipelineTest params) throws IOException {
             args.add("-R");
             args.add(referenceFile.getAbsolutePath());
         }
+        args.add("--"+ MarkDuplicatesSpark.DO_NOT_MARK_UNMAPPED_MATES);
```

Comment:
Do you have tests that cover the case where this argument is not specified?

Reply:
yes, the corresponding tests (with the same names and arguments in AbstractMarkDuplicatesTester)